content/blog/2024/09/25/2024-09-25-update-center-brownouts-4.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Brownout on Update Center (updates.jenkins.io):\n 11 and 12 September 2024"
```

Comment:
Title needs updating, (I’m on mobile otherwise I would)

Reply:
Good catch, thanks! done in https://github.com/jenkins-infra/jenkins.io/commit/effe3f5fd6841f58cd3e10487282232bd4186760, should be visible in a few minutes